AssignmentOne/AssignmentOne/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using AssignmentOne.Array;
+using AssignmentOne.Interface;
+using AssignmentOne.Loops;
+using AssignmentOne.Recursion;
+
+namespace AssignmentOne
+{
+    internal class Program
+    {
+
+        static void Main()
+        {
+          //compressedstring
+            Compressed compressed = new Compressed();
+            compressed.comstr("jjaaggaaddiisshh");
```

Comment:
you should not eliminate the duplicates in the input instead you need to show the count. For example for the about input, output must be j2a2g2a2d2i2s2h2.